File: codingbat/java/warmup-1/posNeg.java
public boolean posNeg(int a, int b, boolean negative) {
  
    if (negative == true && ( a < 0 && b < 0)) {
	return true ;
    }
    if (negative == false && ((a < 0 && b > 0) || (a > 0 && b < 0))) {
	return true ;
    }
    return false ;
}

File: python/codigbat/warmup-1/pos_neg.py
def pos_neg(a, b, negative):
  if (negative == True and a < 0 and b < 0):
    return True 
  if (((a < 0 and b > 0) or (a > 0 and b < 0)) and negative == False):
    return True 
  return False 